x-pack/legacy/plugins/apm/e2e/README.md

```diff
@@ -0,0 +1,76 @@
+# End-To-End (e2e) Test for APM UI
+
+## Ingest static data into Elasticsearch via APM Server
+
+1. Start Elasticsearch and APM Server, using [apm-integration-testing](https://github.com/elastic/apm-integration-testing):
+
+```shell
+$ git clone https://github.com/elastic/apm-integration-testing.git
+$ cd apm-integration-testing
+./scripts/compose.py start master --all --apm-server-record
```

Comment:
`--apm-server-record` that particular flag is not enabled in the CI pipeline, is it required?

Reply:
I updated this flags to match the local setup